### PR TITLE
Tidy up GitHub Actions

### PR DESCRIPTION
**Pull request by ialarmedalien**
# Description of PR purpose/changes

* Updates actions to the most recent versions
* Removes two actions that are not used

# Jira Ticket / Issue #

N/A

### CODE REVIEW

File: .github/codeql.yml [hunk 1]
```diff
@@ -3,8 +3,6 @@ name: "Code scanning - action"
 on:
   push:
   pull_request:
-  schedule:
-    - cron: '0 19 * * 0'
```
Comment (ialarmedalien) [inline .github/codeql.yml:7]: we don't really need a scheduled job - on pull and push is fine IMO

File: .github/workflows/manual_build.yml [hunk 2]
```diff
@@ -1,23 +0,0 @@
-name: Build and Tag Latest Image
```
Comment (ialarmedalien) [inline .github/workflows/manual_build.yml:1]: this was an experimental file but the experiment never went anywhere

File: .github/workflows/pr_build.yml [hunk 3]
```diff
@@ -25,20 +25,20 @@ jobs:
       tags: pr-${{ github.event.number }},latest
     secrets: inherit
   build-main-open:
-    if: (github.base_ref == 'main' || github.base_ref == 'master') && github.event.pull_request.merged == false
+    if: github.base_ref == 'main' && github.event.pull_request.merged == false
```
Comment (ialarmedalien) [inline .github/workflows/pr_build.yml:28]: removing the 'master' since we use 'main'

File: .github/workflows/release-main.yml [hunk 4]
```diff
@@ -2,9 +2,6 @@
 name: Release - Build & Push Image
 on:
   release:
-    branches:
```
Comment (ialarmedalien) [inline .github/workflows/release-main.yml:5]: invalid tag

File: .github/workflows/tag_environments.yaml [hunk 5]
```diff
@@ -1,35 +0,0 @@
----
```
Comment (ialarmedalien) [inline .github/workflows/tag_environments.yaml:1]: unused

File: .github/workflows/tag_environments.yaml [hunk 6]
```diff
@@ -1,35 +0,0 @@
...
-name: Tag Image For Deploy
-on:
-  repository_dispatch
```
Comment (ialarmedalien) [inline .github/workflows/tag_environments.yaml:4]: invalid value

**Review by briehl — APPROVED** [review]

**Comment by briehl** [comment]
Codecov continues to be feisty. I'm ignoring it, as tests passed.

**Pull request merged**